http_logs/challenges/default.json

```diff
@@ -312,4 +328,65 @@
           "clients": 1
         }
       ]
+    },
+    {
+      "name": "append-no-conflicts-index-reindex-only",
+      "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only. After indexing, same data are reindexed",
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
+          }
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "index": "logs-*",
+            "request-params": {
+              "wait_for_status": "{{cluster_health | default('green')}}",
+              "wait_for_no_relocating_shards": "true"
+            }
+          }
+        },
+        {
+          "operation": "index-append",
+          "warmup-time-period": 240,
+          "clients": {{bulk_indexing_clients | default(8)}}
+        },
+        {
+          "name": "refresh-after-index",
+          "operation": "refresh",
+          "clients": 1
+        },
+        {
+          "operation": "force-merge",
+          "clients": 1
+        },
+        {
+          "name": "refresh-after-force-merge",
+          "operation": "refresh",
+          "clients": 1
+        },
+        {
+          "name": "reindex",
+          "operation": {
+            "operation-type": "reindex",
+            "body": {
+              "source": {
+                "index": "logs-*"
+              },
+              "dest": {
+                "index": "reindexed-logs"
+              }
+            },
+            "request_timeout": 7200
```

Comment:
Did you intend to make this operation blocking too? In other words, do you want to add `wait_for_completion=true` so that there is a record of how long this operation takes?

Reply:
One FYI thing to be generally aware of is that if there are proxies involved in the path between Rally and Elasticsearch the proxy may timeout the request abruptly (if `wait_for_completion=true`) and then the Elasticsearch Python client will retry.

Reply:
Reindex defaults to `wait_for_completion=true`, so I think that part is OK?

I am not sure there is anything to do about the proxy issue? Is there maybe a way to instruct the python client to never retry and instead fail (unless we have scenarios where a retry is expected)? That would be a general rally change I think?

Reply:
Ah great, I missed that `true` is the default for `reindex`! I don't think there's anything to be done, I just raised it as a FYI in case you want to run it against an external environment and get surprising results.